src/components/selectbutton/SelectButton.tsx

```diff
@@ -126,12 +127,13 @@ export function SelectButton({
 }
 
 export function MultipleSelect({
+  className,
   status,
   children,
   size,
 }: SelectButtonListProps) {
   return (
-    <StyledSelectButtons status={status} size={size}>
+    <StyledSelectButtons status={status} size={size} className={className}>
```

Comment:
We could add `wrapperProps` or `containerProps` instead of adding single properties, this should be more future-proof (image we want to pass some data-* or aria-* attributes).

```suggestion
export type SelectButtonListProps = {
  containerProps?: React.ComponentProps<typeof StyledSelectButtons>;
  // ...
  
  
  containerProps,
  status,
  children,
  size,
}: SelectButtonListProps) {
  return (
    <StyledSelectButtons status={status} size={size} {...containerProps}>
```